pantheon/src/main/java/tech/pegasys/pantheon/cli/PantheonCommand.java

```diff
@@ -552,6 +552,13 @@ public void run() {
               + "or specify the beneficiary of mining (via --miner-coinbase <Address>)");
     }
 
+    if (permissionsConfigFile != null) {
+      if (!permissionsAccountsEnabled && !permissionsNodesEnabled) {
+        throw new ParameterException(
```

Comment:
Do we want this to be an error? Convention here seems to be options that aren’t going to have an effect is a warning like on line 636.

Reply:
have posed the question for @jakehaugen

Reply:
changed to warning